pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -28,17 +28,21 @@
 PULP_SERVICES = ("pulpcore-content", "pulpcore-api", "pulpcore-worker@1", "pulpcore-worker@2")
 
 
-## Pulp No Leftover Objs Feature
-
-
 def pytest_addoption(parser):
     group = parser.getgroup("pulp-smash")
     group.addoption(
         "--pulp-no-leftovers",
         action="store_true",
         dest="pulp_no_leftovers",
+        default=False,
         help="Enable this to have Pulp plugins check for objects leftover by tests.",
     )
+    group.addoption(
+        "--nightly",
```

Comment:
I guess I'm unclear on what the benefit is from this PR. Running all the tests is what you don't specify filtering through markers.

Reply:
"nightly" is our way of saying "this test will run slow and use a lot of resources". So the benefit is that a contributor running pytest in his dev env will run all the usual tests and get a result in a reasonable amount of time. They can still run all tests by specifying `--nightly`. This is literally the example given in the pytest docs, just they call it "slow".

Reply:
To me it is rather important how to run tests as a dev. Making running the tests locally a bad experience, i think slows down development at the cost of running tests more often in the CI. And the increased load in the CI has all sorts of bad impact on the project. What i want to say is "the default set of tests" to me is what is exercised by simply calling `pytest` without parameters.